Split up

jrleeman opened the pull request:
Initial stab at splitting things up into individual modules. Probably need to discuss naming some. I think this is about the right level of granularity?

dopplershift commented on examples/defining_new_state_relation.py:
```diff
@@ -1,14 +1,14 @@
 import numpy as np
 import matplotlib.pyplot as plt
```
no longer need `plt`

dopplershift commented on examples/slide_hold_slide.py:
```diff
@@ -1,6 +1,6 @@
 import numpy as np
 import matplotlib.pyplot as plt
```
Here too.

dopplershift commented on examples/velocity_step_two_state.py:
```diff
@@ -1,6 +1,6 @@
 import numpy as np
 import matplotlib.pyplot as plt
```
Another one.

dopplershift commented:
I think this is the right granularity. A shorter option for `plotting` would be `plots` or `plot`, but that's up to you.

jrleeman commented:
Thinking about splitting up the tests by module as much as possible. Standard practice?

dopplershift commented:
It's pretty standard to have one test file per module.

jrleeman commented:
Ok. I've changed the naming a tiny bit and updated the examples/tests to reflect that. If it looks good to you, we'll merge and the test split-up can happen in another PR?

dopplershift commented:
:+1:
